elasticdl/python/master/optimizer_wrapper.py

```diff
@@ -491,26 +278,14 @@ def _create_slot_variable_in_optimizer(
 
     def _report_to_kv_store(self):
```

Comment:
`_report_to_kv_store` to `_update_embedding_param`

Reply:
Done